examples/complete/vpc.tf

```diff
@@ -10,7 +10,9 @@ module "tamr_networking" {
   ingress_cidr_blocks                = var.ingress_cidr_blocks
   availability_zones                 = var.availability_zones
   create_public_subnets              = true
-  enable_nat_gateway                 = true
+  enable_nat_gateway                 = false
```

Comment:
Was this changed to prove that we're using the EMR endpoint?

Reply:
Yes.